gcc 5.2.0 shows warnings.

Compiling with `gcc` 5.2.0  in `Linux 4.2.3-1-ARCH` these warnings are shown:

```
src/traceroute/interface.cpp: In member function ‘measurement_kit::traceroute::ProbeResultMeaning measurement_kit::traceroute::ProbeResult::get_meaning()’:
src/traceroute/interface.cpp:77:72: warning: format ‘%d’ expects argument of type ‘int’, but argument 4 has type ‘measurement_kit::traceroute::ProbeResultMeaning’ [-Wformat=]
                            icmp_type, icmp_code, PRM_::GOT_REPLY_PACKET);
                                                                        ^
src/traceroute/interface.cpp:84:40: warning: format ‘%d’ expects argument of type ‘int’, but argument 4 has type ‘measurement_kit::traceroute::ProbeResultMeaning’ [-Wformat=]
                              m->meaning);
                                        ^
src/traceroute/interface.cpp:89:48: warning: format ‘%d’ expects argument of type ‘int’, but argument 4 has type ‘measurement_kit::traceroute::ProbeResultMeaning’ [-Wformat=]
                          icmp_code, PRM_::OTHER);
                                                ^
```

--- a/src/traceroute/interface.cpp
+++ b/src/traceroute/interface.cpp
@@ -73,20 +73,20 @@ static ProbeResultMapping MAPPINGv6[] = {
 
 ProbeResultMeaning ProbeResult::get_meaning() {
     if (valid_reply) {
-        mk::debug("type %d code %d meaning %d (got reply packet)",
-                  icmp_type, icmp_code, PRM_::GOT_REPLY_PACKET);
+        mk::debug("type %d code %d meaning %lld (got reply packet)",
+                  icmp_type, icmp_code, (long long)PRM_::GOT_REPLY_PACKET);
         return PRM_::GOT_REPLY_PACKET;
     }
     for (auto m = is_ipv4 ? &MAPPINGv4[0] : &MAPPINGv6[0];
          m->meaning != PRM_::OTHER; ++m) {
         if (m->type == icmp_type && m->code == icmp_code) {
-            mk::debug("type %d code %d meaning %d", icmp_type,
-                      icmp_code, m->meaning);
+            mk::debug("type %d code %d meaning %lld", icmp_type,
+                      icmp_code, (long long)m->meaning);
             return m->meaning;
         }
     }
-    mk::debug("type %d code %d meaning %d (other)", icmp_type,
-              icmp_code, PRM_::OTHER);
+    mk::debug("type %d code %d meaning %lld (other)", icmp_type,
+              icmp_code, (long long)PRM_::OTHER);
     return PRM_::OTHER;
 }
 

